stories/Atom/Buttons-and-labels/Cta_button/buttons.scss

```diff
@@ -48,27 +53,77 @@
   &.cta--arrow {
     &::after {
       @extend %ctaimg;
-      background: url(#{$img-path-Icon}/arrow-left-expand-defaut.svg) no-repeat left center;
+      background: url(#{$img-path-Icon}/arrow-left-expand-defaut.svg) no-repeat
+        left center;
     }
     &:hover {
       &::after {
-        background: url(#{$img-path-Icon}/arrow-left-expand-hover.svg) no-repeat left center;
+        background: url(#{$img-path-Icon}/arrow-left-expand-hover.svg) no-repeat
+          left center;
         margin-left: $spacing-24;
         transform: translateX(10px);
-        transition: transform 0.2s ease;
+        -moz-transform: translateX(10px);
+        -webkit-transform: translateX(10px);
+        -o-transform: translateX(10px);
+        -ms-transform: translateX(10px);
+        @include arrow-transition;
         width: 100px;
       }
     }
   }
   &.cta--space {
     &::after {
       @extend %ctaimg;
-      background: url(#{$img-path-Icon}/arrow-left-expand-defaut.svg) no-repeat left center;
-      transition: transform 0.2s ease;
+      background: url(#{$img-path-Icon}/arrow-left-expand-defaut.svg) no-repeat
+        left center;
+      @include arrow-transition;
     }
     &:hover {
       &::after {
         transform: translateX(70%);
+        -moz-transform: translateX(70%);
+        -webkit-transform: translateX(70%);
+        -o-transform: translateX(70%);
+        -ms-transform: translateX(70%);
+      }
+    }
+  }
+}
+[dir="rtl"] {
+  .button {
+    &.button--primary {
+      &.button--arrow {
+        &::after {
+          @include transform-rotate;
+          margin-right: 14px;
+          margin-left: 0px;
+        }
+        &:hover::after {
+          @include transform-rotate-50;
+        }
+      }
+    }
+  }
+  .cta__link {
+    &.cta--space,
+    &.cta--arrow {
+      &::after {
+        @include transform-rotate;
+        margin-right: 14px;
+        margin-left: 0px;
+      }
+      &:hover::after {
+        @include transform-rotate-50;
+      }
+    }
+    &.cta--arrow {
+      &:hover::after {
+        transform: translateX(-10%) rotate(180deg);
+        -moz-transform: translateX(-10%) rotate(180deg);
+        -webkit-transform: translateX(-10%) rotate(180deg);
+        -o-transform: translateX(-10%) rotate(180deg);
+        -ms-transform: translateX(-10%) rotate(180deg);
+        @include arrow-transition;
```

Comment:
@pallavikadam-srijan don't use so much nesting in the css and create the mixins for the autoprefixers so that we don't have repeat so many lines